tools.yaml

```diff
@@ -34,3 +34,321 @@ batman.r-forge.r-project.org:
+  url: https://3omics.cmdm.tw
+  author:
+    - name: MDDLab
+  license: 
+    kind: 
+    url:
+
+# paid product
+acdlabs.com/products/adh/spectrusprocessor/index.php:
+  name: ACD/ NMR Processor Academic Edition
+  description: |
+    "Free NMR processing software for academic and non-commercial use only"
+  url: https://www.acdlabs.com/products/adh/spectrusprocessor/index.php
+  author:
+    - name: ACD/Labs
+  license: 
+    kind: 
+    url:
+
+CHEMDATA.NIST.GOV:
+  name: Mass Spectrometry Data Center
+  description: >
+    "Mass spectral libraries and data analysis tools include AMDIS 
+    (Automated Mass Spectral Deconvolution and Identification System for GC/MS) 
+    and MS Interpreter (for fragmentation analysis)."
+  url: https://chemdata.nist.gov
+  author:
+    - name: National Institute of Standards and Technology 
+      email: massspec@nist.gov
+  license: 
+    kind: open ? 
+    url:
+
+cts.fiehnlab.ucdavis.edu/:
+  name: Chemical Translation Service
+  description: >
+    Web-based service for metabolite identifier translation between > 200 common biological databases, including; 
+    Chemical Name, InChIKey, PubChem, ChemSpider, BioCyc, ChEBI, CAS, HMDB, KEGG and LIPID MAPS.
+  url: http://cts.fiehnlab.ucdavis.edu/
+  author: 
+    - name: Sajjan Mehta
+    - name: Diego Pedrosa
+  license: 
+    kind: GNU LESSER GENERAL PUBLIC LICENSE
+    url: http://www.gnu.org/licenses/lgpl-3.0.html
+
+# paid product
+chenomx.com:
+  name: Chenomx NMR Suite
+  description: |
+    "Chenomx specializes in mixture analysis for applications in life sciences such as 
+    metabolomics, food/nutrition, and cell culture research."
+  url: https://www.chenomx.com
+  author: 
+    - name: Chenomx
+  license:
+    kind:
+    url:
+
+cytoscape.org:
+  name: Cytoscape
+  description: |
+    "Cytoscape is an open source software platform for visualizing complex networks and 
+    integrating these with any type of attribute data"
+  url: https://cytoscape.org
+  author: 
+  license: 
+    kind: GNU LESSER GENERAL PUBLIC LICENSE
+    url: http://www.gnu.org/licenses/lgpl-3.0.html
+
+github.com/dgrapov/DeviumWeb:
+  name: DeviumWeb
+  description: |
+    "Interactive multivariate data analysis in R implemented in shiny."
+  url: https://github.com/dgrapov/DeviumWeb
+  author: 
+    - name: Dmitry Grapov
+      email: dgrapov@gmail.com
+  license:
+    kind: AGPLv3
+    url: http://www.tldrlegal.com/l/AGPL3
+
+genome.jp/kegg/:
+  name: Kyoto Encyclopedia of Genes and Genomes
+  description: >
+    "KEGG is a database resource for understanding high-level functions and 
+    utilities of the biological system, such as the cell, the organism 
+    and the ecosystem, from molecular-level information, especially large-scale molecular datasets generated by 
+    genome sequencing and other high-throughput experimental technologies"
+  url: https://www.genome.jp/kegg/
+  author:
+  license:
+    kind: "?"
```

Comment:
From the link, it says there is an Academic license and three Commercial licenses. So change this to Multiple